### ISSUE TITLE
Documentation is outdated

### ISSUE THREAD

**Issue by itsandreramon** [airbnb/MvRx#445]
Documentation is outdated at many places which makes it really hard to follow the examples.

**Comment by gpeal** [comment]
@itsandreramon Are you using 1.0 or 2.0? 2.0 docs are in the docs folder on the release/2.0 branch. They will be hosted once it is live.

**Comment by itsandreramon** [comment]
Currently using 1.5.1. For example this one from the current docs:

```kotlin
data class MyState(val listing: Async<Listing> = Uninitialized) : MvRxState
class MyViewModel(override val initialState: MyState) : MvRxViewModel<MyState>() {
    init {
        fetchListing()
    }

    private fun fetchListing() {
       // ...
    }
}
```

When copied to my IDE it looks like this:
<img src="https://i.imgur.com/r0heRYX.png"/>

**Comment by gpeal** [comment]
@itsandreramon https://github.com/airbnb/MvRx/wiki/Integrating-MvRx-In-Your-App#create-your-own-base-viewmodel

The docs are being completely rewritten for 2.0 (the upgrade should be easy though)

**Comment by itsandreramon** [comment]
Why is this not mentioned? How is one supposed to find this?

**Comment by gpeal** [comment]
@itsandreramon The 2.0 docs will be more clear. Thanks for your input.
